airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/pipeline.py

```diff
@@ -90,10 +90,17 @@ async def run_connectors_pipelines(
         # HACK: This is to get a long running dockerd service to be shared across all the connectors pipelines
         # Using the "normal" service binding leads to restart of dockerd during pipeline run that can cause corrupted docker state
         # See https://github.com/airbytehq/airbyte/issues/27233
-        docker_hub_username_secret = dagger_client.set_secret("DOCKER_HUB_USERNAME", contexts[0].docker_hub_username)
```

Comment:
The issue was that `.docker_hub_username` returned None, which `dagger_client.set_secret` was not a fan of

Reply:
Hence the error raised on L248 of [these logs](https://dagger.cloud/runs/037c6817-f50e-40f3-a194-382e73852bdd)